XVim2/Xcode/SourceEditorViewWrapper.swift

```diff
@@ -59,7 +59,14 @@ import Cocoa
 @_silgen_name("scev_wrapper_call2") func _set_cursor_style(_: UnsafeRawPointer, _: CursorStyle)
 @_silgen_name("scev_wrapper_call3") func _get_data_source(_: UnsafeRawPointer) -> (UnsafeMutableRawPointer)
 @_silgen_name("scev_wrapper_call4") func _set_selected_range(_: UnsafeRawPointer, _: XVimSourceEditorRange, modifiers: Int)
-@_silgen_name("scev_wrapper_call5") func _add_selected_range(_: UnsafeRawPointer, _: XVimSourceEditorRange, modifiers: Int)
+@_silgen_name("scev_wrapper_call5") func _add_selected_range(
+    _: UnsafeRawPointer,
+    _: XVimSourceEditorRange,
+    anchor: XVimSourceEditorRange?,
+    modifiers: Int,
+    scrollPlacement: Any?,
+    alwaysScroll: Bool
+)
```

Comment:
NOTE: `add_selected_range` method now have 6 arguments.
Our assembly implemented code (SourceEditorViewWrapper.s) support up to 8 arguments.